tasmota/tasmota_xdrv_driver/xdrv_57_9_tasmesh.ino

```diff
@@ -649,6 +679,16 @@ void MESHEverySecond(void) {
       MESHsetWifi(1);
       WifiBegin(3, MESH.channel);
     }
+
```

Comment:
Suggestion: send the heartbeat request only if no other packets came in within the last second

Reply:
I think it would be a good idea to send the heartbeat regardless because maybe the packet we sent before didn't make it to the broker (e.g. maybe the TTL was too short)

Reply:
@barbudor Okay, or should be changed?

Reply:
I don't think it's critical, we can leave it like this